src/index.tsx

```diff
@@ -148,3 +112,89 @@ export async function init(config: IClientConfig): Promise<EppoClient> {
 export function getInstance(): EppoClient {
   return EppoReactNativeClient.instance;
 }
+
+export class EppoPrecomputedReactNativeClient extends EppoPrecomputedClient {
+  public static initialized = false;
+
+  public static instance: EppoPrecomputedReactNativeClient =
+    new EppoPrecomputedReactNativeClient({
+      precomputedFlagStore: memoryOnlyPrecomputedFlagsStore,
+      subject: {
+        subjectKey: '',
+        subjectAttributes: {},
+      },
+    });
+}
+
+/**
+ * Initializes the Eppo precomputed client with configuration parameters.
+ * This method should be called once on application startup.
+ * @param config - client configuration
+ * @public
+ */
+export async function precomputedInit(
+  config: IPrecomputedClientConfig
+): Promise<EppoPrecomputedClient> {
+  if (EppoPrecomputedReactNativeClient.initialized) {
+    return EppoPrecomputedReactNativeClient.instance;
+  }
+
+  validation.validateNotBlank(config.apiKey, 'API key required');
+  validation.validateNotBlank(
+    config.precompute.subjectKey,
+    'Subject key required'
+  );
+
+  const {
+    apiKey,
+    precompute: { subjectKey, subjectAttributes = {}, banditActions },
+    baseUrl,
+    requestTimeoutMs,
+    numInitialRequestRetries,
+    numPollRequestRetries,
+    pollingIntervalMs,
+    pollAfterSuccessfulInitialization = false,
+    pollAfterFailedInitialization = false,
+    skipInitialRequest = false,
+  } = config;
+
+  // Set up parameters for requesting updated configurations
+  const requestParameters: PrecomputedFlagsRequestParameters = {
+    apiKey,
+    sdkName,
+    sdkVersion,
+    baseUrl,
+    requestTimeoutMs,
+    numInitialRequestRetries,
+    numPollRequestRetries,
+    pollAfterSuccessfulInitialization,
+    pollAfterFailedInitialization,
+    pollingIntervalMs,
+    throwOnFailedInitialization: true, // always use true here as underlying instance fetch is surrounded by try/catch
+    skipInitialPoll: skipInitialRequest,
+  };
+
+  const subject: Subject = { subjectKey, subjectAttributes };
+
+  EppoPrecomputedReactNativeClient.instance =
+    new EppoPrecomputedReactNativeClient({
+      precomputedFlagStore: memoryOnlyPrecomputedFlagsStore,
+      requestParameters,
+      subject,
+      precomputedBanditStore: memoryOnlyPrecomputedBanditsStore,
+      banditActions,
```

Comment:
I believe this line will now set `banditActions`. @typotter please let me know if this doesn't work for you still